src/css/custom.css

```
@@ -1152,8 +1152,62 @@ body:has(.navbar-sidebar--show) > div:first-of-type {
+  font-style: normal;
+  font-weight: 600;
+  color: var(--surface-brand-darkest-indigo);
+  background: var(--surface-brand-light-indigo);
+  border-radius: 5px;
+  padding: 2px 5px;
+  text-transform: capitalize;
+  line-height: 15px;
+  content: "Feature";
+  top: 50%;
+  transform: translateY(-50%);
+  line-height: 15px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

line-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)